api/envoy/type/matcher/node.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package envoy.type.matcher;
+
+import "envoy/type/matcher/metadata.proto";
+import "envoy/type/matcher/string.proto";
+
+option java_package = "io.envoyproxy.envoy.type.matcher";
+option java_outer_classname = "NodeProto";
+option java_multiple_files = true;
+
+// [#protodoc-title:NodeMatcher]
```

Comment:
nit: "// [#protodoc-title: Node matcher]"

Reply:
I see that other files still have the original format, e.g.: https://github.com/envoyproxy/envoy/blob/master/api/envoy/type/matcher/v3/metadata.proto#L15

Reply:
Do you mind fixing the other one(s)? It's a doc nit but generally we try for human readable docs.